src/Documentation/SidebarMenu/SidebarMenu.js

```diff
@@ -1,13 +1,44 @@
-import React, { Fragment } from 'react'
+import React from 'react'
 import $ from 'jquery'
+import PerfectScrollbar from 'perfect-scrollbar'
 // components
 import DownloadButton from '../../DownloadButton'
-// utils
-import startCase from 'lodash.startcase'
-import includes from 'lodash.includes'
 // styles
 import styled from 'styled-components'
 import { media, OnlyDesktop } from '../../styles'
+// sidebar helpers
+import { getParentsListFromPath } from './helper'
+
+function SidebarMenuItem({ children, label, path, activePaths, onNavigate }) {
+  const isActive = activePaths && activePaths.includes(path)
```

Comment:
use lodash.includes, otherwise we are breaking compatibility

Reply:
You are right, will change.